extensions/amp-a4a/amp-a4a-format.md

```diff
@@ -61,6 +61,19 @@ rel="canonical">` tag.
    and won't be independently search-indexed, so self-referencing
    would be useless.
 
+1. The creative can include optional meta tags in HTML head as identifiers,
+   in the format of `<meta name="amp4ads-id" content="${namespace}:${id}">`.
```

Comment:
As discussed in #7730, `meta` does not allow arbitrary attributes. 

I limited the charset to [a-zA-Z0-9-_] (we can always open up more later), do you know if it is enough for DFP?